website/pages/docs/integrate/nia-integration.mdx

```diff
@@ -0,0 +1,98 @@
+---
+layout: docs
+page_title: Network Infrastructure Automation Integration Program <sup>Tech Preview</sup>
+sidebar_title: NIA Integration Program <sup>Tech Preview</sup>
+description: Guide to partnership integrations for Consul NIA
+---
+
+# Network Infrastructure Automation Integration Program <sup>Tech Preview</sup>
+
+HashiCorp’s Network Infrastructure Integration (NIA) Program allows partners to build integrations that allow customers to automate dynamic application workflows leveraging network and security infrastructure at runtime. Detailed below is the approach to integrate your networking technology along with the clearly defined steps, links to information sources, and checkpoints.
+
+## Network Infrastructure Automation
+
+Network Infrastructure Automation (NIA) relies on a declarative, workflow and service driven network automation architecture. NIA leverages an agent `Consul-Terraform-Sync` which leverages Terraform as the underlying automation tool and utilizes the Terraform provider ecosystem to drive relevant change to the network infrastructure. This usage of the Terraform provider in conjunction with Consul-Terraform-Tasks allows for HashiCorp Consul to act as the core data source where it is updated with the state of the infrastructure.
+
+The Consul-Terraform-Sync agent executes one or more automation tasks with an appropriate value of service variables based on updates from the Consul service catalog. Each task consists of a runbook automation written as a compatible Terraform module using resources and data sources for the underlying network infrastructure. The Consul-Terraform-Sync agent runs on the same node as a Consul agent.
+
+[![NIA Architecture](/img/nia-highlevel-diagram.png)](/img/nia-highlevel-diagram.png)
+
+## NIA Program Steps
+
+The NIA Integration Program has six steps. By following these steps, Consul-Terraform-Sync compatible Terraform modules can be developed. They are then published as “verified” Consul-Terraform-Sync modules on the [NIA page consul.io](https://www.consul.io/use-cases/network-infrastructure-automation).
+
+-> **Note:** A prerequisite to be eligible for NIA Integration program includes having a “verified” provider on Terraform registry for the appropriate technology. Please follow the guidelines to enroll in the Terraform Provider Development Program if you do not presently have a “verified” provider.
+
+[![NIA Integration Program Steps](/img/nia-integration-program.png)](/img/nia-integration-program.png)
```

Comment:
minor: i think it would be nice if this image used consul pink to align with branding on other pages

Reply:
👍 I updated the image.